src/relay/backend/graph_plan_memory.cc

```diff
@@ -82,9 +82,8 @@ class StorageAllocaBaseVisitor : public ExprVisitor {
   void VisitExpr_(const TupleNode* op) final {
     std::vector<StorageToken*> fields;
     for (Expr field : op->fields) {
-      auto tok = GetToken(field);
-      ICHECK_EQ(tok.size(), 1U);
-      fields.push_back(tok[0]);
+      auto tokens = GetToken(field);
```

Comment:
Does this handle 2 levels of nesting?

Reply:
yes, there is `VisitExpr(field)` inside `GetToken`, where nesting is recursively visited and token generated for each tensor in the nest.